5.3.whisperer.py

```diff
@@ -0,0 +1,11 @@
+import re
+def whisperer(local_path):
+    file = open(local_path, 'rb')# open to read as bytes
+    file_content = file.read() 
+    file.close()
+    decode_file = file_content.decode('utf-8', 'ignore') # decode the bytes into string
+    regex = r"([a-z]{5,}!)" # regular expression that matching at least 5 letters and '!' in the end
```

Comment:
I would use documentation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/